scapy/layers/dns.py

```diff
@@ -29,7 +29,7 @@
 from scapy.pton_ntop import inet_ntop, inet_pton
 
 
-def dns_get_str(s, p, pkt=None, _internal=False):
+def dns_get_str(s, p, pkt=None, _fullpacket=False):
```

Comment:
I think that it is a good opportunity to rename this one letter variables.

Reply:
`s` stands for string (pretty well used in Scapy)
`p` stands for pointer

Should I keep `s` and get `p` -> `pointer` (a bit long, it’s used a lot afterwards)

Reply:
If this change is not too much trouble, it will improve the readability of the algorithm,